connect/core/src/main/java/org/camunda/connect/Connectors.java

```diff
@@ -40,7 +40,7 @@ public class Connectors {
   public static String SOAP_HTTP_CONNECTOR_ID = "soap-http-connector";
 
   /** The global instance of the manager */
-  static Connectors INSTANCE = new Connectors();
+  private static final Connectors INSTANCE = new Connectors();
```

Comment:
```suggestion
  static final Connectors INSTANCE = new Connectors();
```

❌ All classes not residing in an `impl` package are public API. We cannot restrict visibility further for backward compatibility reasons.